src/python/WMCore/WMSpec/Steps/Executor.py

```diff
@@ -175,7 +187,9 @@ def setCondorChirpAttrDelayed(self, key, value):
             condor_config_dir = os.path.dirname(condor_config)
             condor_chirp_bin = os.path.join(condor_config_dir, 'main/condor/libexec/condor_chirp')
             if not os.path.isfile(condor_chirp_bin):
-                condor_chirp_bin = None
+                # Singularity container might not have CONDOR_CONFIG env.
+                #TODO It should have fixed from Singularity setting
+                condor_chirp_bin = getFullPath("condor_chirp")
 
         if condor_chirp_bin:
```

Comment:
Per your other comment, maybe we should add

and os.access(condor_chirp_bin, os.X_OK)